astropy/visualization/wcsaxes/core.py

```diff
@@ -289,12 +290,16 @@ def plot_coord(self, *args, **kwargs):
             if isinstance(frame0, SkyCoord):
                 frame0 = frame0.frame
 
+            native_frame = self._transform_pixel2world.frame_out
```

Comment:
This doesn't work on 3.2.2:
```
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <astropy.visualization.wcsaxes.core.WCSAxes object at 0x13297d210>
args = (<SkyCoord (ICRS): (ra, dec) in deg
    (359.76045223, 0.26876217)>, 'o')
kwargs = {'transform': <matplotlib.transforms.CompositeGenericTransform object at 0x131f72190>}
frame0 = <ICRS Coordinate: (ra, dec) in deg
    (359.76045223, 0.26876217)>

    def plot_coord(self, *args, **kwargs):
        """
        Plot `~astropy.coordinates.SkyCoord` or
        `~astropy.coordinates.BaseCoordinateFrame` objects onto the axes.
    
        The first argument to
        :meth:`~astropy.visualization.wcsaxes.WCSAxes.plot_coord` should be a
        coordinate, which will then be converted to the first two parameters to
        `matplotlib.axes.Axes.plot`. All other arguments are the same as
        `matplotlib.axes.Axes.plot`. If not specified a ``transform`` keyword
        argument will be created based on the coordinate.
    
        Parameters
        ----------
        coordinate : `~astropy.coordinates.SkyCoord` or `~astropy.coordinates.BaseCoordinateFrame`
            The coordinate object to plot on the axes. This is converted to the
            first two arguments to `matplotlib.axes.Axes.plot`.
    
        See Also
        --------
    
        matplotlib.axes.Axes.plot : This method is called from this function with all arguments passed to it.
    
        """
    
        if isinstance(args[0], (SkyCoord, BaseCoordinateFrame)):
    
            # Extract the frame from the first argument.
            frame0 = args[0]
            if isinstance(frame0, SkyCoord):
                frame0 = frame0.frame
    
>           native_frame = self._transform_pixel2world.frame_out
E           AttributeError: 'WCSAxes' object has no attribute '_transform_pixel2world'

astropy/visualization/wcsaxes/core.py:295: AttributeError
```

Reply:
(relies on the reorganization from https://github.com/astropy/astropy/pull/8874)